plugins/catalog-backend/src/service/DefaultLocationService.test.ts

```diff
@@ -197,7 +197,7 @@ describe('DefaultLocationServiceTest', () => {
           { type: 'url', target: 'https://backstage.io/catalog-info.yaml' },
           true,
         ),
-      ).rejects.toThrowError('Duplicate nested entity: location:default/foo');
```

Comment:
I think we probably still want to confirm the error message too. Maybe just `.toThrow(/Duplicate nested entity: location:default/foo'/)` should work here?

Reply:
I also wondering how to check the error message with InputError 😅 but didn't found any example.
Ok let me try that.

Reply:
Updated